src/plots/mapbox/mapbox.js

```diff
@@ -138,22 +138,22 @@ proto.createMap = function(calcData, fullLayout, resolve, reject) {
     map.on('moveend', function(eventData) {
         if(!self.map) return;
 
-        var view = self.getView();
-
-        opts._input.center = opts.center = view.center;
-        opts._input.zoom = opts.zoom = view.zoom;
-        opts._input.bearing = opts.bearing = view.bearing;
-        opts._input.pitch = opts.pitch = view.pitch;
```

Comment:
@etpinard because this was getting triggered by `map.setCenter` etc (see existing comment below) we were unnecessarily writing these settings back to `gd.layout` on initial plotting. It doesn't cause any test failures to move these into the protected block below, do you see any issues with this change?

Reply:
I can't think of any potential issues here :ok_hand: 

Just make sure all the `@noCI` mapbox tests still pass.